ct-aa/dotTheory---JavaScript
# ✨ JS-614: Review Response Render

# Related Tickets
- [JS-614](https://codeandtheory.atlassian.net/browse/JS-614)

# Description
Added the **Review Response Render** challenge solutions.

# Validation
* Open the Terminal and go to the directory of the **Review Response Render* folder.
* Run "node app.js".
* Visit http://localhost:3000/ on the browser.
* The browser should display the sample posts.

<img width="1779" alt="Screen Shot 2023-01-03 at 7 56 12 PM" src="https://user-images.githubusercontent.com/93116252/210352772-f8c26b2a-7e43-4569-a387-20781f8f4e1a.png">

# Risks
N/A
